### PR TITLE
Relay [ #metastruct (IRC) <> (Discord) #xchat ]

### PR DESCRIPTION
**Pull request by ghost**
Please setup following environment variables before running:

- DISCORDTOKEN (bot token)
- DISCORDCHANNEL (channel id)
- DISCORDGUILD (guild id)

______
If the variables won't be set up, the bot will not work. (excluding DISCORDTOKEN)

discord_2017-12-23_14-37-57

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -9,6 +9,30 @@
     homepage = "https://metastruct.net"
   ]]
   
+local serverid = os.getenv("DISCORDGUILD")
+local channelid = os.getenv("DISCORDCHANNEL")
+
+if not serverid or not channelid then
```
Comment (Techbot121) [inline init.lua:15]: I think it's ok when we hardcode this

File: init.lua [hunk 2]
```diff
@@ -37,24 +62,43 @@ local wlit = weblit.app
...
 client:on('ready', function()
+	guild = client:getGuild(serverid)
+	channel = guild:getChannel(channelid)
 	print('Logged in as '.. client.user.username)
 end)
 
 client:on('messageCreate', function(message)
+	if message.channel == channel and message.author ~= client.user then
+		if message.content:starts(".") then
+			c:say("#metastruct", "Command call requested by "..message.author.username..":")
```
Comment (Techbot121) [inline init.lua:80]: maybe do a single liner instead of two?

**Comment by Techbot121** [comment]
@Python1320

File: init.lua (hunk 2 above)
Comment (WicopeeDot) [reply init.lua]: uh yes

**Pull request merged**